dev-packages/cli/package.json

```diff
@@ -32,6 +32,12 @@
   "dependencies": {
     "@theia/application-manager": "^0.14.0",
     "@theia/application-package": "^0.14.0",
+    "@types/mkdirp": "^0.5.2",
+    "@types/requestretry": "^1.12.3",
+    "mkdirp": "^0.5.0",
+    "requestretry": "^3.1.0",
+    "tar": "^4.0.0",
```

Comment:
missing @types/tar ?

Reply:
Yes, that's correct. I added it as part of #6933 once I realized it was missing, but I will extract it to a different pull request:
- https://github.com/eclipse-theia/theia/pull/6933/files#diff-84c2900e9405f9a2214fe8a17c59b4d8R37